drivers/dac/ad5754r/ad5754r.c

```diff
@@ -216,6 +216,30 @@ int ad5754r_update_dac_all_ch_registers(struct ad5754r_dev *dev,
 	return no_os_gpio_set_value(dev->gpio_ldac, NO_OS_GPIO_HIGH);
 }
 
+/**
+ * @brief Trigger LDAC.
+ * @param dev - The device structure.
+ * @return 0 in case of success, negative error code otherwise.
+ */
+int ad5754r_ldac_trigger(struct ad5754r_dev *dev)
+{
+	int ret;
+
+	if (!dev)
+		return -EINVAL;
+
+	if (dev->gpio_ldac) {
+		ret = no_os_gpio_set_value(dev->gpio_ldac, NO_OS_GPIO_LOW);
+		if (ret)
+			return ret;
+
```

Comment:
would there be a need for a delay here ?

Reply:
The minimum delay is supposed to be 20ns. I think the overheads involved accumulate to greater than easily. Using the mbed platform, the delay was found to be ~300ns.

Reply:
I don't think there's a function for no-OS function for nsec delay. Should I explicitly add a 1us delay?

Reply:
Let's add the 1us delay, otherwise, somebody may be chasing this issue in the future. It's not guaranteed to happen but if the processor is really fast I think it could happen on xilinx for example or on high end stm32's

Reply:
I've added the 1us delay now